languagetool-language-modules/zh/src/main/resources/META-INF/org/languagetool/language-module.properties

```diff
@@ -1 +1 @@
-languageClasses=org.languagetool.language.Chinese
+languageClasses=org.languagetool.language.Chinese, org.languagetool.language.SimplifiedChinese, org.languagetool.language.TraditionalChinese
```

Comment:
org.languagetool.language.Chinese can probably be removed here, can't it? If everybody always uses Simplified or Traditional, there's no need for it?